src/plugins/intel_cpu/src/nodes/kernels/riscv64/jit_generator.cpp

```diff
@@ -116,20 +126,27 @@ void jit_generator::vfneg_vv(const Xbyak_riscv::VReg& vd, const Xbyak_riscv::VRe
 }
 
 Xbyak_riscv::LMUL jit_generator::float2lmul(const float lmul) {
-    if (lmul == 0.125f)
+    if (lmul == 0.125f) {
```

Comment:
Let's use switch case instead

Reply:
@a-sidorova We have discussed it a bit and Arseniy assumed that the whole 'ratio' logic can be rewritten with 'log ratio', so we deal only with integral types, and not the floating poing ones.
What do you think?